vnext/Microsoft.ReactNative/ReactApplicationDelegate.cpp

```diff
@@ -20,10 +20,24 @@ using namespace Windows::ApplicationModel::Activation;
 
 namespace winrt::Microsoft::ReactNative::implementation {
 
-static void ApplyArguments(ReactNative::ReactNativeHost const & /*host*/, std::wstring const &arguments) noexcept {
-  // Microsoft::ReactNative::implementation::ReactNativeHost* hostImpl {
-  // get_self<Microsoft::ReactNative::implementation::ReactNativeHost>(host)};
+static void ApplyArguments(ReactNative::ReactNativeHost const &host, std::wstring const &arguments) noexcept {
+  Microsoft::ReactNative::implementation::ReactNativeHost *hostImpl{
+      get_self<Microsoft::ReactNative::implementation::ReactNativeHost>(host)};
   if (!arguments.empty() /*&& host.HasInstance()*/) {
+    std::wistringstream argumentStream(arguments);
+    std::wstring token;
+    while (std::getline(argumentStream, token, L' ')) {
+      if (token == L"directDebugging" && std::getline(argumentStream, token, L' ')) {
+        wchar_t *nextPosition = nullptr;
```

Comment:
we should have a -? usage option to describe this and other options we'd like to support

Reply:
I updated the PR. Did you have something like this in mind?